web/styles/zulip.css

```diff
@@ -3478,70 +3382,91 @@ select.invite-as {
     }
 }
 
-#personal-menu-dropdown {
-    display: block;
-    margin: 0;
-    position: relative;
-    padding: 0;
-    border: solid 1px var(--color-border-dropdown-menu);
-    background-color: var(--color-background-dropdown-menu);
-    max-height: 85vh;
-    min-width: 230px;
-    overflow-x: hidden;
-    user-select: none;
-
-    .simplebar-content {
-        min-width: max-content;
-    }
-
-    .inner {
-        border-radius: 6px;
-        overflow: hidden;
+.keyboard-shortcut-option {
+    & span.tooltip-hotkey-hint {
+        margin-left: auto;
+        padding: 0 4px;
+        color: var(--color-hotkey-hint);
+        border-color: var(--color-hotkey-hint);
     }
 }
 
-.personal-menu-tippy {
-    .tippy-box {
-        border: 0;
-    }
+#gear-menu-dropdown {
```

Comment:
It would probably add too much churn to this PR, but it would be great to move this whole chunk into `popovers.css`--especially because of the selector-competition I've noted with this selector and the `navbar-dropdown-menu` class.

Reply:
Nice idea, done.